config/opal_check_package.m4

```diff
@@ -36,24 +36,39 @@ AC_DEFUN([_OPAL_CHECK_PACKAGE_HEADER], [
     unset opal_Header
 
     opal_check_package_header_happy="no"
-    AS_IF([test "$3" = "/usr" || \
+    AS_IF([test "$3" = "" || \
+           test "$3" = "/usr" || \
            test "$3" = "/usr/local"],
            [ # try as is...
             AC_VERBOSE([looking for header without includes])
-            AC_CHECK_HEADERS([$2], [opal_check_package_header_happy="yes"], [])
+            AC_CHECK_HEADERS([$2], [opal_check_package_header_happy="yes"], [], [$6])
```

Comment:
Adding the fourth argument to `AC_CHECK_HEADERS` makes this line disagree with the `AC_VERBOSE` message immediately above, and is certainly wrong if `CPPFLAGS` is not modified.  I don't think this part of the change is good.